src/components/table/ItemCard.tsx

```diff
@@ -77,13 +83,19 @@ const ItemCard = ({
     </Grid2>
   );
 
+  // show link thumbnail
+  let thumbnail = thumbnailUrl;
+  if (!thumbnail && item.type === ItemType.LINK) {
+    thumbnail = getLinkThumbnailUrl(item.extra);
+  }
+
```

Comment:
You could reuse `getImgUrl()` if you decided to use a function in the previous component ;)